core/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -160,26 +160,6 @@ public IndexRequest() {
     }
 
     /**
-     * Copy constructor that creates a new index request that is a copy of the one provided as an argument.
-     * The new request will inherit though headers and context from the original request that caused it.
-     */
-    public IndexRequest(IndexRequest indexRequest) {
```

Comment:
I think I use this in reindex to use one index request to stamp out others. It seemed like a good idea at the time.